WebService.Test/v1/Models/SimulationApiModel/SimulationApiModelTest.cs

```diff
@@ -143,6 +143,28 @@ Microsoft.Azure.IoTSolutions.DeviceSimulation.WebService.v1.Models.SimulationApi
                 .Wait(Constants.TEST_TIMEOUT);
         }
 
+        [Fact, Trait(Constants.TYPE, Constants.UNIT_TEST)]
```

Comment:
missing test for the average messages per second, making sure the result includes decimals for example